themes/hugo-scroll/assets/css/content.scss

```diff
@@ -265,15 +265,23 @@ table {
         }
     }
 
-    tbody>tr:nth-child(even) {
+    /**tbody>tr:nth-child(even) {
         background-color: #fff5;
         background-blend-mode: lighten;
     }
 
     &.plain tbody>tr:nth-child(odd)>td,
     &.plain tbody>tr:nth-child(odd)>th {
         background: transparent;
+    }**/
```

Comment:
@ShravyaD-coder What I meant was just for the organizer section instead of every table, which was fine in the version before. I can merge this PR once changes in this file and `_gen` are reverted

Reply:
@kazewong I have reverted the changes